docs/reference/configuration/mbed_targets.md

```diff
@@ -194,12 +194,19 @@ Enabling `is_disk_virtual` adds delay after flashing firmware binary to make sur
 
 The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `uARM`, `ARMC5`, `ARMC6`, `GCC_ARM` and `IAR`.
 
-If you specify `ARMC5` in `supported_toolchains`, it means the corresponding target supports `Arm Compiler 5.06 update 6`.
+When using `ARM`, `ARMC5`, `ARMC6` for `supported_toolchains`, please note:
 
-If a target lists both `ARMC5` and `ARM` (or `ARMC6`) in `supported_toolchains`, the Arm Compiler 6.11 will be used when compiling with `ARM` option for `--toolchain`.
+- If you specify `ARMC5` in `supported_toolchains`, it means the target supports Arm Compiler 5 (version 5.06 update 6) only, and the build system uses Arm Compiler 5 when you specify `-t ARM` or `-t ARMC5` with `mbed compile` command. 
```

Comment:
If the target supports Arm Compiler 5, will `supported_toolchains` show `ARMC5`, or do you have to manually add that?

Reply:
@AnotherButler - Good question, the changes which Brian is working on will add everything as required and according to these rules. So manual addition required from user or the reader of this documentation.

Reply:
So if I changed it to "If the target supports Arm Compiler 5 (version 5.06 update 6) only, `supported_toolchains` will specify `ARMC5`", would that be correct?

Reply:
@AnotherButler - It is technically correct. But I think the intended audience for this document is someone who is adding or configuring a target, as in the document title. That's why its written as "if you specify.." meaning whoever is adding or configuring a target. But also want that information to be available for anyone, so I created a link from cli-compile.md to here.